src/main/resources/application.properties

```
@@ -0,0 +1,8 @@
+# H2
+spring.h2.console.enabled=true
+# Datasource
+spring.datasource.url=jdbc:h2:file:~/shopping-cart
```

> 먼저 인메모리가 아닌 방식으로 하여 좀 더 실제 서비스 환경과 유사한 상태에서 구현, 테스트를 진행하고 싶었고,
H2에서도 어플리케이션 종료와 관계 없이 유지되는 형식이 가능하다고 해서 사용해보았는데요.

👍 

> application.properties 파일에 설정한 대로 로컬의 home 경로에 H2 DB 파일이 생성된다는 점 양해 부탁드립니다. 🥲 
만약 이 방법이 적절치 않거나 리뷰 진행에 불편함이 있으시다면 MySQL DB로 변경하도록 하겠습니다!

home 경로에 생성하지 않고 프로젝트 폴더 내에 생성해도 됐을 것 같아요! 물론 도이가 git push 하실 땐 도이의 파일이 올라가지 않도록 git ignore 해줘야겠네요 🙂 
```suggestion
spring.datasource.url=jdbc:h2:file:./db/shopping-cart
```

질문이 하나 있는데요. MySQL DB로 변경하면 어떤 게 달라지나요..?

---

아 경로를 프로젝트 폴더 내로 설정해주면 되는 문제였네요 ㅎㅎㅎ 감사합니다 반영하겠습니다!!
MySQL 같은 다른 DB는 해당 프로그램 관련 디렉토리에 저장되도록 기본 설정이 되어있어서 더 낫겠지? 라고 생각했는데 
사실 파일 저장 위치는 어떤 벤더사이든 설정할 수 있는 부분일 것 같아서 제가 잘못 말씀 드린 것 같아요..!!